cw_bitcoin/lib/electrum_wallet.dart

```diff
@@ -898,6 +918,10 @@ class EstimatedTxResult {
 }
 
 BitcoinBaseAddress addressTypeFromStr(String address, BasedUtxoNetwork network) {
+  try {
+    return BitcoinCashAddress(address).baseAddress;
```

Comment:
let's put the possibility of the address being able to be parsed to bitcoin cash address while it's not a bitcoin cash address, so let's be on the safe side and also add a check if the network is bitcoin cash